deeppavlov/core/agent_v2/state_schema.py

```diff
@@ -13,6 +14,33 @@ def to_dict(self):
                 'user_type': self.user_type,
                 'personality': self.personality}
 
+    @classmethod
+    def get_or_create(cls, *args, **kwargs):
+        """
+        gets or creates an object from init specification
+
+        Args:
+            *args:
+            kwargs:
+
+        Returns: tuple (instance:User, is_created:bool)
+            or raises Exception if Multiple instances are found
+        """
+        results = cls.objects(*args, **kwargs)
+        if results:
+            if len(results) > 1:
+                # raise Exception
```

Comment:
Could you please remove the comments? It seems like they were made for debugging purposes.

Reply:
fixed